CMakeLists.txt

```diff
@@ -147,6 +147,20 @@ IF("${IBAMR_FORCE_BUNDLED_MUPARSER}" STREQUAL "")
   SET(IBAMR_FORCE_BUNDLED_MUPARSER "OFF")
 ENDIF()
 
+#
+# Check whether we have certain functions avaiable
+#
+CHECK_CXX_SOURCE_COMPILES(
+	"
+	#include <memory>
+	int main()
+	{
+	  auto ptr = std::make_unique<int>(42);
+	  return 0;
+	}
+	"
+	IBAMR_HAVE_MAKE_UNIQUE)
```

Comment:
Lets not do this - we should just use our own version of this function everywhere.